src/main/java/io/sealights/plugins/sealightsjenkins/BeginAnalysis.java

```diff
@@ -741,12 +718,6 @@ private void setGlobalConfiguration(Logger logger, SeaLightsPluginInfo slInfo, P
 
         // set proxy
         String proxy = (String) additionalProps.get("proxy");
-        if (StringUtils.isNullOrEmpty(proxy)) {
-            proxy = override_proxy;
```

Comment:
Why did you remove this code?
Did you move the 'proxy' field to the "additional agruments"?